### PR TITLE
fix: Clickable "Upcoming Launches" cards #460 + updates to launchpad [👀]

### LINKED ISSUE #460: Clickable "Upcoming Launches" cards
image

- Type: UX fix
- Frequency: Everytime
- Link: https://app.teritori.com/launchpad
- Description: Make upcoming drops cards clickable to access mint page or ELSE, twitter page

### PR DESCRIPTION
**Pull request by omniwired**
Fixes #460 

- [x] Order by date
- [ ] make upcoming cards link == airtable cell link (could be evolving from Twitter link to mint page link) OR WAY BETTER create a template of mint page when the mint details are not fully confirmed yet so it's available onclick even tho the contract is not deployed yet (ie: https://magiceden.io/launchpad/the_lowlifes).
- [x] check the "% of mint" calculation discrepancy between the open mint page and the project card (id: T R A.I. P P E D == 6% on card & 7% on the mint page).
- [x] fix "ended" status for ETH mint 
- [x] fix the "ended" status for "Teritori Name Service" by removing it from the list (leave it on the marketplace page).

### Notes;

- It needs an update in Netify to use the preview link.
- I made the upcoming cards to point at Twitter, but not the whole new page. 
- I think we should get new designs and implement them in another PR

## Screenshots Mar 2nd

image

image

### CODE REVIEW

File: packages/components/CollectionView.tsx [hunk 1]
```diff
@@ -15,24 +14,35 @@ import { GradientText } from "./gradientText";
...
   const navigateToCollection = useNavigateToCollection(item.id, {
     forceSecondaryDuringMint: item.secondaryDuringMint,
     forceLinkToMint: linkToMint,
   });
   const sizedStyles = useMemo(() => StyleSheet.flatten(styles[size]), [size]);
 
+  const navigateToTwitter = () => {
+    Linking.openURL(item.twitterUrl);
+  };
+  const { info } = useCollectionInfo(item.id);
```
Comment (omniwired) [inline packages/components/CollectionView.tsx:35]: @n0izn0iz what would be the best way to batch these requests?
Do I need to make a new endpoint in the backend?

**Comment by WaDadidou** [comment]
Are you sure about this new collections layouting ?
If yes, can you use `CollectionGallery` into `MarketplaceScreen` also ?

**Review by sakul-budhathoki — APPROVED** [review]
LGTM

File: packages/components/CollectionView.tsx [hunk 2]
```diff
@@ -15,24 +13,42 @@ import { GradientText } from "./gradientText";
...
   });
   const sizedStyles = useMemo(() => StyleSheet.flatten(styles[size]), [size]);
+  let maxSupply;
+  const navigateToTwitter = () => {
+    Linking.openURL(item.twitterUrl);
+  };
+  const { info } = useCollectionInfo(item.id);
+  const percentageMinted = info
+    ? Math.round(
+        (parseInt(info.mintedAmount as string, 10) * 100) /
```
Comment (hthieu1110) [inline packages/components/CollectionView.tsx:37]: use can shortcut this `+info.mintedAmount`

File: packages/components/CollectionView.tsx [hunk 3]
```diff
@@ -15,24 +13,42 @@ import { GradientText } from "./gradientText";
...
+          parseInt(info.maxSupply as string, 10)
+      )
+    : NaN;
+  if (!info) {
+    maxSupply = item.maxSupply;
+  } else {
+    maxSupply = info.maxSupply;
+  }
```
Comment (hthieu1110) [inline packages/components/CollectionView.tsx:45]: you can shortcut this: `maxSupply = info ? info.maxSupply : item.maxSupply`

Comment (omniwired) [reply packages/components/CollectionView.tsx]: done

**Comment by omniwired** [comment]
> Are you sure about this new collections layouting ? If yes, can you use `CollectionGallery` into `MarketplaceScreen` also ?

it will be part of another effort, not this PR

**Review by hthieu1110 — APPROVED** [review]
LGTM

**Comment by zxxma** [comment]
LGTM 👍 (Finally!)

**Pull request merged**